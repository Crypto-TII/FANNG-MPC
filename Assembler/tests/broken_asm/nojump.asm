stms s0, 8201 # 4
jmp 2
stms s0, 8210 # 6
