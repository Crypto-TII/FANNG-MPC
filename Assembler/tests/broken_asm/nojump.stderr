error: a jump jumped beyond the program's boundaries
 --> tests/broken_asm/nojump.asm:2:1
  |
2 | jmp 2
  | ^^^^^ jumped from here
  |